lib/structures/Message.js

```diff
@@ -291,8 +292,15 @@ class Message extends Base {
                 };
             });
         }
-        if(data.stickers !== undefined) {
-            this.stickers = data.stickers;
+        if(data.sticker_items !== undefined) {
+            this.stickerItems = data.sticker_items.map((sticker) => {
+                if(sticker.user) {
+                    sticker.user = client.users.update(sticker.user, client);
+                }
+
+                return sticker;
+            });
+            this.stickers = this.stickerItems;
```

Comment:
Please still use the deprecated `data.stickers`, it shouldn't be changed completely to match another reference for backwards compatibility/preventing a breaking change